index.html

```diff
@@ -224,21 +243,32 @@ <h4>Skills</h3>
         </section>
```

Comment:
- [ ] After adjusting the paragraph you will see that the "Get my resume" overlaps the `Frameworks`, kindly fix it by adding `margin: Xpx Ypx;` in the `.other-section` class to be similar to the Figma design.
- [ ] Ass required by the last CR, please correct the margins of the cards by adding the margin attribute to the `.other-section` class to be similar to the Figma design 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://github.com/Wils-FOURR-speed/My-portfolio/assets/58642949/5d8ce11c-817e-4c5f-8d8d-0ad4036f3124) | ![image](https://github.com/Wils-FOURR-speed/My-portfolio/assets/58642949/5e8e85fe-37ba-4223-b189-1ba1f83d5b45) |